src/renderMain.py

```diff
@@ -31,7 +31,7 @@ def startWindow(toRender, isOnline, fileName):
 
 def renderText(TRtext, ITR, TTR, C):
     from tkinter.font import Font
-    textSizes = ["<katanaclassbig>", 15, "<h1",23.9, "<h2", 17.9, "<h3", 15.9, "<h4", 14, "<h5", 9.9, "<h6", 8, "<p", 12]
+    textSizes = ["<katanaclassbig>", 15, "<katanaclassbig2>",20,"<katanaclassbig3>",30, "<h1",23.9, "<h2", 17.9, "<h3", 15.9, "<h4", 14, "<h5", 9.9, "<h6", 8, "<p", 12]
```

Comment:
this is to allow more customization for sizes

Reply:
We should see if we can come up with some more original names. Maybe "big", "large", and "extralarge"? But the idea of having more font sizes without css is a good one.